### ISSUE TITLE
Sand Swept Tomb Hashr Glitch

### ISSUE THREAD

**Issue by nathanielz** [Wynncraft/Issues#8183]
I have been trying to level, and i am currently level 45. I had some keys left for the sand swept tomb so I decided to use them and when i did i lost each time because of a glitch. Whenever I reached the boss it would just keep re-spawning it after I kill it. i would go for up to about 3 and die from his rush attack thing. So if you could fix this please do because it makes it basically impossible for the level range of the dungeon.

**Comment by LordHelmchen98** [comment]
So you killed the final boss, but it didn't teleport you out?

**Comment by nathanielz** [comment]
Yes several times. Both of them, it kept spawning 2 of them at once, I was
on my 3rd or 4th time killing both.
